src/stickerbook.js

```diff
@@ -610,13 +610,19 @@ class Stickerbook {
       angle: options.rotation
     });
     this.state.sticker.setCoords();
+      
+    var hasBorders = this._config.stickerControls.hasBorders;
```

Comment:
Let's put this inside of the `if` block below, which'll add error checking around `this._config.stickerControls` existing.